### DISCUSSION TITLE
Trumbowyg missing icons

### DISCUSSION THREAD

**Discussion by kevinsmgov** [OrchardCMS/OrchardCore#11979] [Q&A]
When I add "indent" and "outdent" buttons to the Trumbowyg configuration, the buttons appear and function in the bar but there are no icons in them (they are blank). Does anyone know what additional configuration in necessary to make the icons show in the buttons?

**Comment by hishamco** [comment]
Please add a GIF screenshot showing the issue

**Reply by kevinsmgov** [reply]
![Screenshot 2022-07-09 080233](https://user-images.githubusercontent.com/5798763/178111293-cc4a1b8a-58d8-4c31-9cd9-33f70c4bfa6f.gif)
